examples/impute/plot_missing_values.py

```diff
@@ -32,10 +32,22 @@
 from sklearn.impute import SimpleImputer, IterativeImputer, MissingIndicator
 from sklearn.model_selection import cross_val_score
 
+# Importing warnings library
+import warnings
+from sklearn.exceptions import ConvergenceWarning
+
+warnings.filterwarnings("ignore",
```

Comment:
We should avoid having to do that inside the example. Users might not understand what this is doing. It is better to fix the parameters to fix the convergence warning instead.